README.md

```diff
@@ -105,6 +106,22 @@ Run the program
 make run
 ```
 
+### MongoDB
+
+```zsh
+export DB_MODE=mongodb
+```
+
+Setup env variables
+
+- ATLAS_URI: Connection string to MongoDB Atlas
```

Comment:
Change the env name to MONGO_URI
In future, maybe you want to host your own mongodb, hence the name ATLAS doesn't make sense here. It conflicts with the purpose of practicing Clean Architecture in this project.